### PR TITLE
Port more docs to TQL2

### PR DESCRIPTION
**Pull request by mavam**
The amount of to-be-ported docs is getting smaller, day by day...

### CODE REVIEW

**Comment by mavam** [comment]
Note to the reviewer: this is how the deprecated reference looks like right now:

image

File: web/docs/operators/apply.md
Comment (dominiklohmann) [inline web/docs/operators/apply.md:1]: This is not in the TQL2 docs.

File: web/docs/operators/show.md
Comment (dominiklohmann) [inline web/docs/operators/show.md:1]: There is no replacement for `show operators` and `show serves` in TQL2.

**Review by dominiklohmann — APPROVED** [review]

File: web/docs/operators/show.md
Comment (mavam) [reply web/docs/operators/show.md:1]: We decided that we won't port `show serves` as it's primarily a debugging vehicle.

**Pull request merged**